src/index.js

```diff
@@ -109,10 +110,40 @@
           $(this).attr('data-selecting', 'selecting');
           plugin.$el.find('.time-slot').attr('data-disabled', 'disabled');
           plugin.$el.find('.time-slot[data-day="' + day + '"]').removeAttr('data-disabled');
+
+          bodyElement.mouseleave(function(event) {
```

Comment:
I am hesitant to add behaviors that are outside of the plugin itself (in this case, window scrolling) and would love to keep this plugin super lightweight. Do you think we can achieve the feature by exposing another event like `selecting`(similar to the only event [`selected`](https://github.com/artsy/day-schedule-selector/blob/96e39e3fd5dc466d19a5f0dcd53b0d5944fac07a/src/index.js#L119)) that clients can listen and handle additional behaviors?